xtask/src/bootstrap/yarn.rs

```diff
@@ -37,7 +37,6 @@ impl Bootstrap for YarnCmd {
         let mut cmd = Command::new("yarn");
         run_cmd(
             cmd.current_dir(repository_root()?)
-                .arg("--no-lockfile")
```

Comment:
It might be good to use `--frozen-lockfile` which is usually recommended for CI, I believe, because in some cases `yarn install` can cause updates even if a lock file exists. Not a blocker though.